llama_hub/web/whole_site/base.py

```diff
@@ -0,0 +1,129 @@
+from typing import List
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+from selenium.common.exceptions import WebDriverException
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+import chromedriver_autoinstaller
```

Comment:
Can we lazily import these, so that we don't need to make additions to the `test_requirements`?

As an example:

https://github.com/run-llama/llama-hub/blob/364feb5b9c41a081da53aae02e894c6caa2f478d/llama_hub/llama_packs/gradio_agent_chat/base.py#L36

Reply:
We can do that for chromedriver_autoinstaller, but for selenium, it might be a bit slow if we lazy import at different places of the functions. What do you think?